RealEstate/apps/core/templates/core/categories.html

```diff
@@ -124,7 +84,7 @@ <h2 class="modal-title">Edit your category!</h2>
         <form action="" role="form" method="post">
         	{% bootstrap_form editForm%}
         	{% csrf_token %}
-        	<div style="margin-top: 40px">
+        	<div id="saveButtonMargin">
```

Comment:
I think this `<div>` doesn't really give us much. Put the style on the button and let's delete these.